src/locales/atlassian_missing_french.json

```diff
@@ -3,5 +3,8 @@
     "fabric.editor.blockquote2": "Citation",
     "fabric.editor.remove": "Supprimer",
     "fabric.editor.linkPlaceholder": "Coller le lien",
-    "fabric.editor.linkAddress": "Adresse du lien"
+    "fabric.editor.linkAddress": "Adresse du lien",
+    "fabric.editor.layoutWide": "Élargir",
+    "fabric.editor.layoutFullWidth": "Pleine largeur",
+    "fabric.editor.layoutFixedWidth": "Retour au centre"
```

Comment:
Nit: on peut faire des PRs chez eux pour rajouter ça ?

Reply:
"Note: This component is designed for internal Atlassian development.
External contributors will be able to use this component but will not be able to submit issues." 🤷‍♂️ 

I've a task somewhere to post on their forum to ask them how we can enhance the french translation.